src/main/java/com/faforever/client/fx/PlatformService.java

```diff
@@ -44,9 +45,19 @@ public void showDocument(String url) {
   /**
    * Show a file in its parent directory, if possible selecting the file (not possible on all platforms).
    */
-  
+  @SneakyThrows
   public void reveal(Path path) {
-    noCatch(() -> show(path.toFile()));
+    if (isWindows) {
+      Path finalPath = path;
```

Comment:
why do you need a another reference on `path`?

Reply:
Lambdas in java can only access final or effictively final local variables